staging/src/k8s.io/apiserver/pkg/apis/example/v1/types.go

```diff
@@ -55,11 +55,11 @@ type Pod struct {
 // state of a system.
```

Comment:
I made additional changes in this file because descriptions are about pods, so I though it would fit